alerts/resource_alerts.libsonnet

```diff
@@ -43,9 +43,9 @@
             },
           } else {
             expr: |||
-              sum(namespace_cpu:kube_pod_container_resource_requests:sum{%(ignoringOverprovisionedWorkloadSelector)s}) - (sum(kube_node_status_allocatable{resource="cpu"}) - max(kube_node_status_allocatable{resource="cpu"})) > 0
+              sum(namespace_cpu:kube_pod_container_resource_requests:sum{%(ignoringOverprovisionedWorkloadSelector)s}) - (sum(kube_node_status_allocatable{resource="cpu", %(kubeStateMetricsSelector)s}) - max(kube_node_status_allocatable{resource="cpu"})) > 0
```

Comment:
```suggestion
              sum(namespace_cpu:kube_pod_container_resource_requests:sum{%(ignoringOverprovisionedWorkloadSelector)s}) - (sum(kube_node_status_allocatable{resource="cpu", %(kubeStateMetricsSelector)s}) - max(kube_node_status_allocatable{resource="cpu", %(kubeStateMetricsSelector)s})) > 0
```

IIUC, shouldn't this be for all instances of KSM's metrics?

Reply:
it makes no difference with `max()` if you have duplicate metrics or not but it does with `sum()`.

Reply:
For the sake of maintaining uniformity throughout the expressions, and to support KSM selectors at all expected instances, I still think we should make this change.

Reply:
thanks @rexagod, make sense- done